Source/Examples/ExampleLibrary/Utilities/PlotModelUtilities.cs

```diff
@@ -301,24 +301,38 @@ public static PlotModel Transpose(this PlotModel model)
 
             foreach (var series in model.Series.OfType<XYAxisSeries>())
             {
-                if (series.XAxis != null && series.XAxisKey == null)
+                if (series.XAxisKey == null)
```

Comment:
Would it be possible to simplify this by calling `EnsureAxes` on the series before otherwise interrogating it?

Reply:
I like the idea in general, but this is not available here to use, as it is `protected internal`...

Reply:
Yep, sorry.